coremltools/converters/caffe/_caffe_converter.py

```diff
@@ -178,7 +178,7 @@ def convert(model, image_input_names=[], is_bgr=False,
 
     """
     from ...models import MLModel
-    from ...models.utils import convert_neural_network_weights_to_fp16 as convert_neural_network_weights_to_fp16
+    from ...models.utils import _convert_neural_network_weights_to_fp16 as _convert_neural_network_weights_to_fp16
```

Comment:
The `as _convert_neural_network_weights_to_fp16` is redundant. It's not doing anything for you.

Reply:
Right corrected it